src/pages/academy/grading/subcomponents/GradingWorkspace.tsx

```diff
@@ -312,17 +316,32 @@ const GradingWorkspace: React.FC<Props> = props => {
               grading!.answers[questionId].student.name,
               grading!.answers[questionId].team
             )}
+            studentAnswer={
+              grading!.answers[questionId].question.type === 'programming'
+                ? grading!.answers[questionId].question.answer
+                : 'N/A'
+            }
             studentUsernames={
               grading!.answers[questionId].student.username
                 ? [grading!.answers[questionId].student.username]
                 : grading!.answers[questionId].team!.map(member => member.username)
             }
+            is_llm={!!llm_grading && grading!.answers[questionId].question.type == 'programming'}
+            llm_course_level_prompt={llm_course_level_prompt || ''}
+            llm_assessment_prompt={grading!.assessment.llm_assessment_prompt}
+            llm_question_prompt={
+              (grading!.answers[questionId].question.type === 'programming' &&
+                grading!.answers[questionId].question.llm_prompt) ||
+              ''
```

Comment:
All of this shouldn't be in the frontend to avoid prompt injection. Just use the value (that is already stored in the backend anyway)

Reply:
Discussed, will be changing formats to a format that does not require explicit declaration of each type of prompt